src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -169,20 +174,51 @@ class Validator_30 : public AbstractValidator {
         bool exception = true) const override;
 
     bool IsPaddingSupported() const override;
+
+    bool ValidateCnn1D(const std::string& name, const uint32_t inHeight, const uint32_t inWidth,
+    const uint32_t inChannels, const uint32_t kH, const uint32_t kW, const uint32_t kN,
+    const uint32_t strideH, const uint32_t strideW, const uint32_t dilationH, const uint32_t dilationW,
+    OvGnaType inPrecision, bool exception = true) const override;
 };
 
 class Validator_35 : public AbstractValidator {
-    static const RangeLimit2D kInputHWLimit;
-    static const RangeLimit kInputChannelsNumberLimit1B;
-    static const RangeLimit kInputChannelsNumberLimit2B;
-
-    static const RangeLimit kKernelNumberLimit;
-    static const RangeLimit2D kKerneHWlLimit;
-    static const RangeLimit2D kStrideHWLimit;
-    static const RangeLimit2D kDilationLimit;
-
-    static const RangeLimit2D kPoolingWindowHWLimit;
-    static const RangeLimit2D kPoolingStrideHWLimit;
+    struct CnnLimits {
```

Comment:
Can this structure be used in upcoming validators or it is specific only for 3.5?